pkg/operation/botanist/component/nginxingress/nginxingress.go

```diff
@@ -308,6 +308,11 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 					Resources: []string{"ingressclasses"},
 					Verbs:     []string{"get", "list", "watch"},
 				},
+				{
+					APIGroups: []string{"coordination.k8s.io"},
+					Resources: []string{"leases"},
+					Verbs:     []string{"list", "watch"},
+				},
```

Comment:
This is being added for all Kubernetes versions whereas it is required only for Kubernetes version >=1.22.

Reply:
True, but I don't consider this an issue (we already do it similarly for other resources, e.g. https://github.com/gardener/gardener/blob/b93eeced331603a6527d71555fa5d05093852b42/pkg/operation/botanist/component/nginxingress/nginxingress.go#L306-L310)